docs/RELEASE.md

```diff
@@ -0,0 +1,175 @@
+# Release process
+
+The release process for nobl9-go cannot be discussed without the context of
+our platform releases. In the following document, `n9` will refer to our
```

Comment:
avoid using words like our, us, make it more formal and precise, i.e. say 'the context of Nobl9 platform releases' instead